proto/prysm/v1alpha1/validator-client/web_api.proto

```diff
@@ -168,10 +168,10 @@ service Auth {
     }
 }
 
-// Type of key manager for the wallet, either direct, derived, or remote.
+// Type of key manager for the wallet, either derived, local, or remote.
 enum KeymanagerKind {
     DERIVED = 0;
-    IMPORTED = 1;
+    LOCAL = 1;
```

Comment:
One issue with naming it local is that derived is also local...
either imported keys vs derived from a seed I believe
,a recovered key

Reply:
resolved based on discord conversation. a subsequent PR will remove the derived type and move the mnemonic functionality into the LOCAL keymanager type.